configure.ac

```diff
@@ -686,6 +686,22 @@ AS_IF([test "$host" = "$build"], [
 AC_SUBST([PYTHON_CFLAGS])
 AC_SUBST([PYTHON_LIBS])
 
+AC_PATH_PROGS([PYTHON], [ \
+    python \
```

Comment:
I think we should put `python` after the `python3...` variants, so python3 is preferred (`python` is normally python2...)

Reply:
So do I, will fix that in a bit, but this will break the build on some platforms due to missing python3-pytest.